### PR TITLE
[Community Pipeline] FRESCO: Spatial-Temporal Correspondence for Zero-Shot Video Translation

### PR DESCRIPTION
**Pull request by SingleZombie**
# What does this PR do?

Implement a comminuty pipeline. Related issue: #7408 

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [x] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [x] Was this discussed/approved via a GitHub issue or the [forum](https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63)? Please add a link to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/diffusers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

@sayakpaul @yiyixuxu @DN6

### CODE REVIEW

File: examples/community/README.md [hunk 1]
```diff
@@ -4035,6 +4036,96 @@ onestep_image = pipe(prompt, num_inference_steps=1).images[0]
...
+    control_frames,
+    num_inference_steps=20,
+    strength=0.75,
+    controlnet_conditioning_scale=0.7,
+    generator=generator,
+    negative_prompt=n_prompt
+).images
+
+export_to_video(
+    output_frames, output_video_path, 5)
```
Comment (sayakpaul) [inline examples/community/README.md:4125]: Let's also link the outputs as a GIF maybe so that it renders nicely?

**Review by sayakpaul — APPROVED** [review]
Thanks for this pipeline! I just left a few minor comments.

**Comment by SingleZombie** [comment]
> Thanks for this pipeline! I just left a few minor comments.

Good idea! I have edited the example script.

**Review by sayakpaul — APPROVED** [review]
Thanks!

**Comment by sayakpaul** [comment]
Thanks for your contributions!

**Pull request merged**